app.go

```diff
@@ -29,16 +28,18 @@ type App struct {
 // tom: added "sslmode=disable" to connection string
 func (a *App) Initialize(user, password, dbname string) {
 	connectionString :=
-		fmt.Sprintf("user=%s password=%s dbname=%s sslmode=disable", user, password, dbname)
```

Comment:
@dandzoit 
I have made this change because I was continuously getting SSL mode should be enable when an API is trying get data from database